spec/pluralrules.html

```diff
@@ -152,12 +150,12 @@
         1. If isFinite(_n_) is *false*, then
           1. Return *"other"*.
         1. If the _pluralRules_.[[MinimumSignificantDigits]] is not *undefined* and _pluralRules_.[[MaximumSignificantDigits]] is not *undefined*, then
-          1. Let _n_ be ToRawPrecision(_n_, _pluralRules_.[[MinimumSignificantDigits]], _pluralRules_.[[MaximumSignificantDigits]]).
+          1. Let _s_ be ToRawPrecision(_n_, _pluralRules_.[[MinimumSignificantDigits]], _pluralRules_.[[MaximumSignificantDigits]]).
         1. Else,
-          1. Let _n_ be ToRawFixed(_n_, _pluralRules_.[[MinimumIntegerDigits]], _pluralRules_.[[MinimumFractionDigits]], _pluralRules_.[[MaximumFractionDigits]]).
+          1. Let _s_ be ToRawFixed(_n_, _pluralRules_.[[MinimumIntegerDigits]], _pluralRules_.[[MinimumFractionDigits]], _pluralRules_.[[MaximumFractionDigits]]).
         1. Let _locale_ be _pluralRules_.[[Locale]].
         1. Let _type_ be _pluralRules_.[[Type]].
-        1. Let _operands_ be ? _GetOperands_(_n_).
+        1. Let _operands_ be ? _GetOperands_(_s_).
```

Comment:
No, we need GetOperands to be called with the string output of `ToRawPrecision` or `ToRawFixed`, so that the options are taken into account. We know that _s_ is here a proper string representation of a number; if it's cast to number and then re-stringified with `ToString` inside GetOperands, the trailing zeroes are lost.